packages/datagateway-search/src/state/reducers/dgsearch.reducer.test.tsx

```diff
@@ -81,7 +81,7 @@ describe('dgsearch reducer', () => {
   });
 
   it('should set maxNumResults property when configuring action is sent', () => {
```

Comment:
Should add an equivalent test for `minNumResults`

Reply:
I also updated dgsearch reducer because `loadMinNumResults` action wasn't handled.